arch/risc-v/src/mpfs/mpfs_serial.c

```diff
@@ -330,6 +364,7 @@ static struct up_dev_s g_uart4priv =
   .parity    = CONFIG_UART4_PARITY,
   .bits      = CONFIG_UART4_BITS,
   .stopbits2 = CONFIG_UART4_2STOP,
+  .fpga      = false,
```

Comment:
I think these can be fpga too. We won't use hw+fpga uarts at the same time. @jlaitine can confirm.

Reply:
Maybe make this selectable via kconfig, so we can get the hw blocks working if needed.

Reply:
Yes, I already started with some stuff, but decided it got too complicated. Single switch looks the best (until you find out you need one MSS UART :)  )

Reply:
Better now, even fpga uart0 works as a console. Waiting for the other uarts, offsets are just ok only for the FPGA UART0, others will be changed once we know what they are. Will add 2 more uarts (now only 6).